src/rustdoc/proposals-and-stabilization.md

```diff
@@ -0,0 +1,72 @@
+# Proposals, Approvals and Stabilization
+It is very common to need to gather feedback and approval when contributing to rustdoc, either
+for permission to proceed with an experiment or refactoring, or when stabilizing a feature. This
+document aims to summarise the various processes that the rustdoc team has for making approval
+decisions and when each should be used.
+
+## Approvals
+There are three mechanisms that the team can use to approve a proposal (not all approval mechanisms
+are suitable for each method of making a proposal - see below):
+
+- r+
+  - A proposal is r+'d when it is approved to be merged.
+  - r+ can only be used to approve a PR.
+- Seconding
+  - A proposal is second'ed when a team member formally endorses the proposal. It is intended that
+    seconding only occur once discussion has concluded and team members have had time to raise
+    concerns. Seconding tentatively accepts a proposal subject to a ten-day waiting period for
+    other team members to raise any concerns.
```

Comment:
Sorry that I wasn't clear, with https://github.com/rust-lang/rust-forge/pull/852#discussion_r2100551695 (removing mention of MCPs) I meant removing the *entire* section about "Seconding". As the now removed sentence "Seconding can only be used to approve a MCP." states, seconding is MCP-specific and only exists in the context of MCPs.

```suggestion
```

Reply:
Makes more sense now. :+1: